config/checker/io.vavr.astub

```diff
@@ -12,34 +12,43 @@ import org.checkerframework.framework.qual.EnsuresQualifierIf;
 package io.vavr;
 
 interface Value<T> {
-  @PolyNull @Pure T getOrElse(@PolyNull T other);
+  @Pure
+  @PolyNull T getOrElse(@PolyNull T other);
 
-  @Nullable @Pure T getOrNull();
+  @Pure
+  @Nullable T getOrNull();
 }
 
 
 package io.vavr.collection;
 
 interface Array<T> {
-  @Pure T get(@IndexFor("this") int index);
+  @Pure
+  T get(@IndexFor("this") int index);
 }
 
 interface List<T> {
-  @Pure T get(@IndexFor("this") int index);
+  @Pure
+  T get(@IndexFor("this") int index);
 
-  @LengthOf("this") @Pure int length();
+  @EnsuresMinLenIf(expression="this", result=false, targetValue=1)
+  @Pure
+  boolean isEmpty();
 
-  @EnsuresMinLenIf(expression="this", result=false, targetValue=1) @Pure boolean isEmpty();
+  @Pure
+  @LengthOf("this") int length();
 }
 
 interface Map<K, V> {
   @PolyNull V getOrElse(K key, @PolyNull V defaultValue);
 }
 
 interface Traversable<T> {
-  @LengthOf("this") @Pure int length();
+  @Pure
```

Comment:
Method itself is `@Pure`, but the result type is `@LengthOf("this") int`; same reasoning applied in other places as well